.happy/terraform/modules/schema_migration/main.tf

```diff
@@ -389,7 +379,7 @@ resource aws_sfn_state_machine sfn_schema_migration {
             },
             "ItemsPath": "$.datasets",
             "Next": "CollectionPublish",
-            "MaxConcurrency": 40,
+            "MaxConcurrency": 32,
```

Comment:
I see there's a MaxConcurrency [here](https://github.com/chanzuckerberg/single-cell-data-portal/pull/5360/files#diff-f4d94af2de6b76dcfe8833d56b096305952f2409ac0c0f1a2b6b0a7a20a67fe6R401) as well set to 40, I assume we should decrease this as well?

Reply:
That is for the collectionSpan which the machines will be able to support 40+ jobs.